### ISSUE TITLE
Scrolling issue on Android

### ISSUE THREAD

**Issue by m4szymanski** [xcarpentier/rn-pdf-reader-js#43]
Hi,
I'm having a strange issue with the latest version (0.2.1) on Android when I try to scroll down I see the following:

![screenshot at feb 12 16-51-22](https://user-images.githubusercontent.com/13990469/52670419-7a41d000-2ee6-11e9-9a9c-6e3a7372aa24.png)

The first page of the PDF loads properly but when I try scrolling up/down to see the next page it doesn't work and instead I see the text "Loading ..." over a grey background, the only way to get to the next page is to press the up/down arrows (which seem to work properly)

Any suggestions?

Thanks!

**Comment by robputer** [comment]
I just tested and have the same issue.  Buttons work fine to navigate, though.  Here is my code for loading:

```
render() {
    return (
        <Container>
            <CHeader title="PDF" press={() => this.props.navigation.toggleDrawer()}/>
            <PdfReader source={{ uri: Expo.FileSystem.documentDirectory + 'test.pdf'}}/>
        </Container>
    );
}
```

**Comment by chasemac** [comment]
All of my pages on Android just show the loading screen...

**Comment by bendada3** [comment]
I have same problem + when I zoom 3-4 times it fall the app :/ any solution?

**Comment by ChiuMungZitAlexander** [comment]
Same here. Pages can be displayed by using arrow down but not if I scroll down.

**Comment by zerbinidamata** [comment]
Hey guys, any solutions on this?

**Comment by boris-burgos** [comment]
I has been researching about this and looks like a incomplete cache implementation.
In react-pdf/Reader.jsx  always show the loading text:

`{!cached && this.renderLoader()}`
 
To fix this issue simply remove this line until the cache system is finished.

**Comment by m4szymanski** [comment]
Hi @boris-burgos & @xcarpentier - sorry if this is a dumb question but is there anything else I need to do here? I grabbed your latest commit (which has that line of code removed) but I am still having the same issue

**Comment by boris-burgos** [comment]
Hi @m4szymanski.

You need to rebuild the pdf reader bundle.

If you like you can use the v0.2.2 release of our fork. That release has two fixes:
- iOs not showing loading icon (#45)
- This issue.

And the pdf bundle is rebuild.

If  you want to do it yourself go to react-pdf folder and execute `npm run build`. Then in the root folder execute `npm run bundle`.

Regards.

**Comment by frankiesardo** [comment]
Nice on @boris-burgos

**Comment by boris-burgos** [comment]
The fork is not published in npm, we don't want to create a "real" public fork.
You can add it with the command:
npm install --save "Codeko/rn-pdf-reader-js#v0.2.2"

**Comment by frankiesardo** [comment]
Thanks. I've tried your version: the loading disappeared but somehow the pdf pages are not displaying (I can only see the header 1/6 and empty screen)

**Comment by frankiesardo** [comment]
Actually: it's even weirder than that: as I click up and down arrow sometimes they are displayed and sometimes not

**Comment by m4szymanski** [comment]
Thanks @boris-burgos ! Your solution worked for me

**Comment by xcarpentier** [comment]
see latest version maybe: 0.2.3

**Comment by frankiesardo** [comment]
I tried 0.2.3 and the same problem persists: at first the content id not displayed (although I can see the number of pages correctly at the top `1/6` `2/6` etc) and after clicking the up and down arrows a few times the content appears and disappears at apparently random intervals

**Comment by bendada3** [comment]
any news? I rly dont want to detach :/

**Comment by azuy212** [comment]
Still having the same issue on latest version (0.2.3) as well. Also persist on master branch too.

**Comment by wiadev** [comment]
Add `"rn-pdf-reader-js": "https://github.com/wiadev/rn-pdf-reader-js/archive/0.2.6.tar.gz"` in package.json

This works fine

**Comment by diegogalocha** [comment]
Thanks @wiadev !! It's working perfectly for me in android.

**Comment by BrickmanRodrigo** [comment]
@wiadev 's solution works perfectly on my end for android too.

**Comment by dhcmega** [comment]
Hi,
I this working with SDK 34 too? I can't even open the PDF file.
Thanks!

**Comment by diegogalocha** [comment]
> Hi,
> I this working with SDK 34 too? I can't even open the PDF file.
> Thanks!

I'm using "rn-pdf-reader-js": "https://github.com/wiadev/rn-pdf-reader-js/archive/0.2.6.tar.gz" with the SDK 34 and it's working perfectly. If you need more information, i can show to you.

**Comment by dhcmega** [comment]
Thanks a lot, I will test it and let you know if I run into problems. Thank you.

**Comment by xcarpentier** [comment]
Should be fixed on `v0.5.1`.
